src/main/java/net/glowstone/block/blocktype/BlockDirectDrops.java

```diff
@@ -16,29 +16,63 @@
 
     private final MaterialMatcher neededTool;
 
+    /**
+     * Creates an instance.
+     * @param dropType the type of item to drop when broken
+     * @param data the damage or block-data value of the dropped item
+     * @param amount the amount to drop
+     * @param neededTool the tool(s) that can break this block
+     */
     public BlockDirectDrops(Material dropType, int data, int amount, MaterialMatcher neededTool) {
```

Comment:
I think the description here should be "Creates a block that drops an item directly when broken"

Reply:
Done.